krator/Cargo.toml

```diff
@@ -19,8 +19,8 @@ maintenance = { status = "actively-developed" }
 
 [features]
 default = ["kube-native-tls"]
-kube-native-tls = ["kube/native-tls", "kube-runtime/native-tls"]
```

Comment:
Why is this no longer needed?

Reply:
Oops good catch. I only meant to remove kube-runtime/native-tls as that feature no longer exists

Reply:
Oh wait, I did keep it. That is why I removed it. That feature doesn't exist anymore (in fact there are no features on kube-runtime anymore)